networkit/test/test_graph.py

```diff
@@ -254,6 +292,41 @@ def testAddEdgeWithMissingNodes(self):
 		self.assertEqual(G.numberOfNodes(), 3)
 		self.assertEqual(G.numberOfEdges(), 2)
 
+	def testRestoreNode(self):
+		G=nk.Graph(2)
+		G.removeNode(1)
+		G.restoreNode(1)
+		self.assertEqual(G.numberOfNodes(), 2)
+
+	def testEdgeWeight(self):
```

Comment:
The name is too generic, and you are testing different behaviors here. Please split this test in more minimal and targeted tests. For example, one test to test `increaseWeight`, another one to test `swapEdge`, another one for `totalEdgeWeight`, etc.